src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
 public class WorkWithFile {
+    private static final int ARRAY_VALUE_INDEX = 1;
+    private static final int ARRAY_NAME_INDEX = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String data = readFile(fromFileName);
+        String report = generateReport(data);
+        writeToFile(toFileName, report);
```

Comment:
This method seems to be doing more than one task which are reading from a file, generating a report, and writing to a file. Consider breaking down this method into smaller ones, each responsible for one task according to the 'Divide program logic' principle.